src/lib/packet.h

```diff
@@ -355,6 +355,7 @@ typedef enum {
     PGP_PKA_RESERVED_DH = 21,             /* Reserved for Diffie-Hellman
                                            * (X9.42, as defined for
                                            * IETF-S/MIME) */
+    PGP_PKA_EDDSA = 22,                   /* EdDSA */
```

Comment:
I'll update the references but EdDSA is basically set at 22 already by virtue of GPG already shipping EdDSA support using that codepoint.